lib/backend/k8s/k8s_fv_test.go

```diff
@@ -747,7 +747,7 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 	}()
 
 	It("should not error on unsupported List() calls", func() {
-		objs, err := c.List(model.BGPPeerListOptions{})
+		objs, err := c.List(model.NodeBGPPeerListOptions{})
```

Comment:
I'll do that as part of the next PR.  It hadn't slipped my notice that this would soon fail.